src/config/default/config.rs

```diff
@@ -74,12 +78,15 @@ impl ConfigStructure for AppConfig {
 
 impl std::default::Default for AppConfig {
     fn default() -> Self {
+        let home = std::env::var("HOME").unwrap();
```

Comment:
These default values can be made into a function like `default_max_preview_size` and used here.
Then it can also be used with `#[serde(default = "func")]`.

Ideally, the default path should be where the rest of the config is; ie. `$XDG_CONFIG_HOME/joshuto/bookmarks.toml`